jlm/llvm/opt/alias-analyses/Andersen.cpp

```diff
@@ -24,6 +24,254 @@ IsOrContainsPointerType(const rvsdg::type & type)
   return IsOrContains<PointerType>(type);
 }
 
+/**
+ * Class collecting statistics from a pass of Andersen's alias analysis
+ */
+class Andersen::Statistics final : public jlm::util::Statistics
```

Comment:
I got a little bit confused that you did not have explicit Stop*() methods for every Start*() method. It took me a while to understand what was going on.

Reply:
It looks better to have corresponding start and stop functions, so I changed it